### PR TITLE
Login signup using Formik

### PR DESCRIPTION
**Pull request by irajbhinde**
Hey! Please review my code & let me know if any changes are to be made

Functionalities Added -

1. Login Page 
2. Signup Page

Files to check - 

- [ ] src/Components/context/auth-context.js
- [ ] src/Components/login_signup/LoginCard.jsx
- [ ] src/Components/login_signup/SignupCard.jsx
- [ ] src/pages/Signup Page/signup-page.css
- [ ] src/pages/login/login-page.css

auth_babytunes

### CODE REVIEW

File: src/Components/login_signup/LoginCard.jsx [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+          <label htmlFor="password" className="input-password">
+            <p>Password*</p>
+            <input
+              id="password"
+              name="password"
+              value={formik.values.password}
+              onChange={formik.handleChange}
+              type="password"
+            />
+            {passType === "text" ? (
```
Comment (SRUJANA-PENUGONDA13) [inline src/Components/login_signup/LoginCard.jsx:75]: This logic is not working can you please check once

File: src/Components/login_signup/LoginCard.jsx [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+              <label htmlFor="remember-me">Remember Me</label>
+            </span>
+            <p className="cursor-pointer">Forgot your password ?</p>
+          </div>
+          <button type="submit" className="login-btn">
+            Login
+          </button>
+          <button
+            type="submit"
+            onClick={() => {
```
Comment (SRUJANA-PENUGONDA13) [inline src/Components/login_signup/LoginCard.jsx:107]: Login as guest also not working in preview

**Review by SRUJANA-PENUGONDA13 — COMMENTED** [review]
I have added few comments. Can you please go through it. and one more suggestion It would be more user friendly if you use different icons for sign in in signup in navbar.

Remaining all looks good to me

File: src/Components/login_signup/LoginCard.jsx [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+                className="signup-pass-icon fa-solid fa-eye-slash"
+              ></i>
+            )}
+          </label>
+          <div className="input-container flex_r">
+            <span className="remember-me-checkbox flex_r">
+              <input type="checkbox" name="" id="remember-me" />
+              <label htmlFor="remember-me">Remember Me</label>
+            </span>
+            <p className="cursor-pointer">Forgot your password ?</p>
```
Comment (akash-1318) [inline src/Components/login_signup/LoginCard.jsx:100]: Remove forgot password and remember me if you not gonna provide functionality for these.

**Review by akash-1318 — COMMENTED** [review]
Just added a small suggestion besides that the login as guest and signup is not working look into this issue and other then that code is well written with keeping all good practices in mind.

**Pull request merged**